api/prometheus.go

```diff
@@ -249,27 +389,209 @@ func (m Metrics) PrometheusMetric() []prometheus.Metric {
 func (wr WalletResponse) PrometheusMetric() []prometheus.Metric {
 	return []prometheus.Metric{
 		{
-			Name: "hostd_wallet",
+			Name: "hostd_wallet_scan_height",
 			Labels: map[string]any{
 				"address": wr.Address,
 			},
-			Value:     float64(wr.ScanHeight),
-			Timestamp: time.Now(),
+			Value: float64(wr.ScanHeight),
 		},
 		{
-			Name:      "hostd_wallet_spendable",
-			Value:     wr.Spendable.Siacoins(),
-			Timestamp: time.Now(),
+			Name: "hostd_wallet_spendable",
+			Labels: map[string]any{
+				"address": wr.Address,
```

Comment:
Is duplicating the address label necessary on all of these?

Reply:
yes because if I have multiple hostd's running with different wallet I would want to delineate each balance type by the hosts